src/reconcile/rec_visibility.c

```diff
@@ -280,6 +346,13 @@ __rec_validate_upd_chain(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_UPDATE *s
     if (!F_ISSET(r, WT_REC_CHECKPOINT_RUNNING))
         return (0);
 
+    /*
+     * Cannot write an update also in the history store to the data store when checkpoint is running
+     * because we cannot delete the history store value.
+     */
+    if (F_ISSET(select_upd, WT_UPDATE_TO_DELETE_FROM_HS))
+        return (EBUSY);
```

Comment:
Will this cause an entire reconciliation to fail, or this update chain to be skipped?

Reply:
The entire reconciliation will fail similar to the case that we detect an out of order update.

Reply:
Can you add stats to track how many evictions can fail with this error? In case there are too many, this limitation can cause performance problems.